deltachat-ios/View/ContactCell.swift

```diff
@@ -327,7 +327,7 @@ class ContactCell: UITableViewCell {
         }
 
         contactRequest.isHidden = !isContactRequest
-        mutedIndicator.isHidden = !isMuted
+        mutedIndicator.isHidden = !isChatMuted
```

Comment:
What about `isAccountMuted` here?

Reply:
idea is to show the "mute icon"  for explicit action the user did - otherwise just all chats would get the icon. this is also consisten with android/desktop

the greyed badges otoh are a more subtile, alos know  not to result from user action. 

i used them here as it otherwise feels wrong to have "blue badge" on the chats that are not summed up on account switcher icon, chats icon, app icon as normally.

i an using this approch since some days and it feels quite intuitive, also what j got from my  testers nearby :)

(we can consider doing the same on android/desktop, however, there the "sum up badges" are leas of a thing, so, let's test out on ios)